docroot/modules/humsci/hs_paragraph_types/hs_paragraph_types.install

```diff
@@ -7,8 +7,25 @@
 
 use Drupal\Core\Site\Settings;
 use Drupal\Core\Config\FileStorage;
+use Drupal\editor\Entity\Editor;
 use Drupal\field\Entity\FieldConfig;
 use Drupal\field\Entity\FieldStorageConfig;
+use Drupal\filter\Entity\FilterFormat;
+
+/**
+ * Implements hook_update_dependencies().
+ */
+function hs_paragraph_types_update_dependencies() {
```

Comment:
This is not needed. The sites are already on D10 with permissions correctly set.

Reply:
Fixed

Reply:
Fixed